wiki/Game_mode/osu!/en.md

```diff
@@ -1,92 +1,98 @@
----
-needs_cleanup: true  # https://github.com/ppy/osu-wiki/issues/9918, also, strange descriptions in many places and unhelpful order if you're trying to see what this game mode is all about.
----
-
 # osu! (game mode)
 
 *For other uses, see [osu! (disambiguation)](/wiki/Disambiguation/osu!).*
 
-![Gameplay of osu!](/wiki/shared/osu-gameplay.jpg "osu! Interface")
+**osu!**, formerly known as *osu!standard*, is the default game mode in which players are required to aim for and click circles using their *cursor* in tune with the rhythm.
```

Comment:
*formerly known as what??**

It was never ever officially known as this, so I'm not sure what this is intending to describe.